bin/travis-build.bash

```diff
@@ -21,7 +21,7 @@ fi
 
 python setup.py develop
 
-if [ -f requirements-py2.txt ]
+if [ -f requirements-py2.txt ] && [ $CKANVERSION != 'master' ]
```

Comment:
this should be testing the python version not the ckan version, right?

Reply:
Completely agree